pyServer/AzureDiskInspectService.py

```diff
@@ -8,7 +8,7 @@
 import socketserver
 import subprocess
 import threading
-import logging
+import logging as pylogging
```

Comment:
I feel like pylogging is not descriptive enough. Can we calls this something like 'pythonLogging'?

Reply:
"py" is kind of  widely used alias for python references. Even many libraries' naming starts from py for python